rubocop.yml

```diff
@@ -13,9 +13,12 @@ Lint/AmbiguousOperator:
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 Lint/AssignmentInCondition:
-  AllowSafeAssignment: false
+  AllowSafeAssignment: true
 Lint/HandleExceptions:
   Enabled: false
+Metrics/AbcSize:
+  Enabled: true
+  Max: 25
```

Comment:
same story like with `MethodLength` - too intrusive & not doable in some projects. though I think this is a better metric than `MethodLength`, I'd like to keep these kind of things out of our RuboCop config and keep the discussion on how to split methods/classes in PRs. humans are better than machines at this :)